crossdc-commons/src/main/java/org/apache/solr/crossdc/common/KafkaCrossDcConf.java

```diff
@@ -16,25 +16,49 @@
  */
 package org.apache.solr.crossdc.common;
 
+import org.apache.kafka.clients.producer.ProducerConfig;
+
 public class KafkaCrossDcConf extends CrossDcConf {
+
+    public static final String DEFAULT_BATCH_SIZE_BYTES = "512000";
```

Comment:
Do these need to be String ?

Reply:
All other places have the same pattern. Should we change all of these out in another PR ?

Reply:
Not necessarily, they are mainly String because there first use is as the default in System.getProperty,which takes a String. That doesn't use Integer.get to pull the property because Null is needed as the default instead of something like -1.

In general it would be good to simplify this config into more útil code. It's a lot of effort to add one currently. I haven't wanted to do it as part of adding more yet though, probably better as a self contained issue.